.github/workflows/ci.yml

```diff
@@ -1,53 +1,35 @@
-# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
-
 name: UI Kit Actions
-
 on:
-  push:
-    branches: [main]
   pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
❓ Is there a reason why you list these types explicitly here? It seems like these three are the default after some quick googling: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
I was googling in the first place because I needed to know what the synchronize event is: https://github.com/orgs/community/discussions/24567

Reply:
Glad you've mentioned this, I wasn't sure myself. I was testing this out and going off of this example - https://d2iq.atlassian.net/wiki/spaces/ENG/pages/88834215/Migration+to+Github+Actions#Code-Checks 
@clintonmedbery do you know if there is any reason to set these explicitly?

Reply:
>  By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. To trigger workflows by different activity types, use the types keyword. For more 

Read through the docs some more. 
Based on this I'll remove specifying this here and in DKP.

Reply:
I like the explicit declarations so I know what I am looking at without having to hunt down documentation, but for me it is not a deal breaker wither way.